connectivity/drivers/cellular/Samsung/TARGET_S5JS100/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+if(${MBED_TOOLCHAIN} STREQUAL "ARM")
+
+	find_library(MODEM_LIBRARY
+		NAMES libSAMSUNG_S5JS100_Modem_Proxy.ar
+		HINTS "./TOOLCHAIN_ARM_STD"
+	)
+elseif(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    find_library(MODEM_LIBRARY
+		NAMES libSAMSUNG_S5JS100_Modem_Proxy.a
+		HINTS "./TOOLCHAIN_GCC_ARM"
+	)
+endif()
+
+
+target_include_directories(mbed-cellular
+    INTERFACE
+        ./RIL/include
+        ./modem/include
+)
+
+
+target_sources(mbed-cellular
+    INTERFACE
+    modem/src/modem_io_device.cpp
```

Comment:
please align these by 4 spaces to the right

Reply:
OK, we will to change it.